.travis.yml

```diff
@@ -72,8 +72,10 @@ script:
     - if [ "$FLAVOR" == "cs" ]; then make DESTDIR=$HOME install; fi
     - if [ "$FLAVOR" == "cs" ]; then cd ..; fi
     - if [ "$FLAVOR" == "cs" ]; then export BOND_COMPILER_PATH=$HOME/usr/local/bin; fi
-    - if [ "$FLAVOR" == "cs" ]; then xbuild /p:Configuration=Debug cs/cs.sln; fi
-    - if [ "$FLAVOR" == "cs" ]; then xbuild /p:Configuration=Fields cs/cs.sln; fi
+    # exclude the Compiler.vcxproj from cs/cs.sln on Linux (xbuild ignore it with a warning but msbuild fail on it)
+    - if [ "$FLAVOR" == "cs" ]; then sed '/21E175D5-BBDD-4B63-8FB7-38899BF2F9D1/d' cs/cs.sln > cs/cs_no_cpp.sln; fi
+    - if [ "$FLAVOR" == "cs" ]; then msbuild /p:Configuration=Debug cs/cs_no_cpp.sln; fi
```

Comment:
Switching to msbuild is understandable, but shouldn't be part of this PR. Can you revert the xbuild to msbuild changes?

Reply:
xbuild didn't get the [property function](https://docs.microsoft.com/en-us/visualstudio/msbuild/property-functions) work fine, so I have to switch to msbuild here. I'll make another PR for the msbuild switching.

Reply:
@chwarr Please see PR #633